gamemodes/terrortown/gamemode/server/sv_corpse.lua

```diff
@@ -181,6 +161,51 @@ function CORPSE.IdentifyBody(ply, rag, searchUID)
 
 		net.Broadcast()
 	end
+
+	if GetConVar("ttt2_confirm_killlist"):GetBool() then
+		-- Handle kill list
+		local ragKills = rag.kills
+
+		local killnicks = {}
+		for i = 1, #ragKills do
+			local victimSIDs = ragKills[i]
+
+			-- filter out disconnected (and bots !)
+			local vic = player.GetBySteamID64(victimSIDs)
+
+			-- is this an unconfirmed dead?
+			if not IsValid(vic) or vic:TTT2NETGetBool("body_found", false) then continue end
+
+			killnicks[#killnicks + 1] = vic:Nick()
+
+			vic:ConfirmPlayer(false)
+
+			-- however, do not mark body as found. This lets players find the
+			-- body later and get the benefits of that
+			--local vicrag = vic.server_ragdoll
+			--CORPSE.SetFound(vicrag, true)
+		end
+
+		if #killnicks == 1 then
+			LANG.Msg("body_confirm_one", {finder = finder, victim = killnicks[1]})
+		elseif #killnicks > 1 then
+			if #killnicks >= 3 then
+				table.sort(killnicks, CorpseSorting)
+			end
```

Comment:
```suggestion
			table.sort(killnicks, CorpseSorting)
```
Can we also inline the sorting function here, since we do not need it elsewhere and it technically does sort nicknames and not corpses

Reply:
redeclaring a function within the loop scope would hurt perf pretty unnecessarily

Reply:
there is no loop and this is negligible

Reply:
Please still inline the function